From f49c9365dece3a5ef86ed3d4801288b533a555e7 Mon Sep 17 00:00:00 2001
From: Alex Vonduar <alexvonduar@msn.com>
Date: Thu, 4 Jul 2019 11:55:16 +0800
Subject: [PATCH] disable NAND, QSPI and Secure boot to save fsbl size

---
 xfsbl_config.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/xfsbl_config.h b/xfsbl_config.h
index ecf4795..2fae302 100644
--- a/xfsbl_config.h
+++ b/xfsbl_config.h
@@ -131,10 +131,10 @@ extern "C" {
  *     - FSBL_FORCE_ENC_EXCLUDE_VAL Forcing encryption for every partition
  *       when ENC only bit is blown will be excluded.
  */
-#define FSBL_NAND_EXCLUDE_VAL			(0U)
-#define FSBL_QSPI_EXCLUDE_VAL			(0U)
+#define FSBL_NAND_EXCLUDE_VAL			(1U)
+#define FSBL_QSPI_EXCLUDE_VAL			(1U)
 #define FSBL_SD_EXCLUDE_VAL			(0U)
-#define FSBL_SECURE_EXCLUDE_VAL			(0U)
+#define FSBL_SECURE_EXCLUDE_VAL			(1U)
 #define FSBL_BS_EXCLUDE_VAL				(0U)
 #define FSBL_EARLY_HANDOFF_EXCLUDE_VAL	(1U)
 #define FSBL_WDT_EXCLUDE_VAL			(0U)
-- 
2.20.1

